vcpkg_windows.txt

```diff
@@ -0,0 +1,5 @@
+boost-build:x86-windows
```

Comment:
why do we need boost build explicitly listed?

Reply:
in my experience, when running 'vcpkg install boost-math:x86-windows', it failed asking me to install 'boost-build:x86-windows' as well. And doing this failed with same error: 'vcpkg install boost-build:x86-windows boost-math:x86-windows'. I had to run first the installation of boost-build:x86-windows, and then run the installation of boost-math:x86-windows separately to succeed.

Reply:
this might be a issue worth reporting in vcpkg, but I'm not sure.